wiki/en/Running-a-Server.md

```diff
@@ -49,22 +48,233 @@ A typical jam might have 4 people, for which you would need 200 Kbit/s * 4 = 800
+
+
+# Bandwidth use
+
+## Audio bandwidth
+
+The audio settings have an impact on the required network bandwidth. The table below summarises network requirements with respect to the configuration of:
+* Channels : stereo/mono
+* Quality : high/medium/low
+* Audio buffer duration : 2.67 ms, 5.33 ms, 10.67 ms, 21.33 ms
+
+With the following units
+* ms : milliseconds
+* Kbit/s : Kilo-bits per second (Reminder : 1 Mbit/s = 1024 Kbit/s, 1 KByte/s = 8 Kbit/s)
+* Mbit/s : Mega-bits per second
+
+| Channels  | Quality | Bandwidth (for buffer : 2.67 ms) |  Bandwidth (for buffer : 5.33 ms) | Bandwidth (for buffer : 10.67 ms) | Bandwidth (for buffer : 21.33 ms) |
+| --------- | ------ | -------- | -------- | -------- | -------- |
+| Stereo    | High   | 894 Kbit/s | 657 Kbit/s | 541 Kbit/s | 483 Kbit/s |
+| Stereo    | Medium | 672 Kbit/s | 444 Kbit/s | 328 Kbit/s | 270 Kbit/s |
+| Stereo    | Low    | 606 Kbit/s | 372 Kbit/s | 256 Kbit/s | 198 Kbit/s |
+| Mono      | High   | 672 Kbit/s | 444 Kbit/s | 328 Kbit/s | 270 Kbit/s 8|
+| Mono      | Medium | 594 Kbit/s | 366 Kbit/s | 250 Kbit/s | 192 Kbit/s |
+| Mono      | Low    | 534 Kbit/s | 306 Kbit/s | 190 Kbit/s | 132 Kbit/s |
+
+## Network bandwidth
+
+There is one upstream (musician sending to the server) and one downstream (server sending back the mix to the musician)
+<figure>
+	<img src="{{site.url}}/assets/img/en-screenshots/bandwidth-diagram.png" loading="lazy" alt="A diagram of Jamulus network bandwidths from different audio qualities ranging from low to high">
+<figcaption>Calculate bandwidth use </figcaption>
+</figure>
+
+Note also that mean ADSL2 transfer rate is 10 Mbit/s for downstream and 1 Mbit/s for upstream. The actual performance depends on distance to the provider, which may [theoretically range from 24 Mbit/s at 0.3 km to 1.5 Mbit/s at 5.2 km](https://en.wikipedia.org/wiki/Asymmetric_digital_subscriber_line) for download rate.
+
+
+
+# Starting a server
+
+
+* **Windows users** - Use the "Jamulus Server" icon in the Windows Start menu.
+
+* **macOS users** - Double-click the "Jamulus Server" icon in Applications (assuming you put the files from the install there as per [these instructions](Installation-for-Macintosh)).
+
+* **Linux users** 
+
+If you have installed the Jamulus client, you can run it in server mode by running Jamulus with the `-s` option as follows:
+
+1. Open a terminal window (`CTRL+ALT+t` on Ubuntu and related distros).
+1. Assuming Jamulus is in `/usr/local/bin`, type `jamulus -s`
+
+Hit return and you should see the server control window. You can stop the server by closing the server window, or by using `CTRL+C` in the terminal.
```

Comment:
I'll correct that path then, since it's wrong.
No icon (or launcher) for server mode yet though it seems. Maybe add that to the package then to bring it into line with other platforms.

~~~
 dpkg-query -L jamulus
/.
/usr
/usr/bin
/usr/bin/jamulus
/usr/share
/usr/share/applications
/usr/share/applications/jamulus.desktop
/usr/share/doc
/usr/share/doc/jamulus
/usr/share/doc/jamulus/changelog.Debian.gz
/usr/share/doc/jamulus/copyright
/usr/share/icons
/usr/share/icons/hicolor
/usr/share/icons/hicolor/512x512
/usr/share/icons/hicolor/512x512/apps
/usr/share/icons/hicolor/512x512/apps/jamulus.png
~~~

Reply:
I will open an issue.